### PR TITLE
[Feature-8021][Document] Add example and notice about task type Spark

### PR DESCRIPTION
**Pull request by QuakeWang**
close: https://github.com/apache/dolphinscheduler/issues/8021

I have updated the document about spark task.

PTAL @zhongjiajie

### CODE REVIEW

File: docs/en-us/2.0.3/user_doc/guide/task/spark.md [hunk 1]
```diff
@@ -1,22 +1,62 @@
...
+- **Descriptive information**: describe the function of the node.
+- **Task priority**: When the number of worker threads is insufficient, they are executed in order from high to low, and when the priority is the same, they are executed according to the first-in first-out principle.
+- **Worker grouping**: Tasks are assigned to the machines of the worker group to execute. If Default is selected, a worker machine will be randomly selected for execution.
+- **Environment Name**: Configure the environment name in which to run the script.
+- **Number of failed retry attempts**: The number of times the task failed to be resubmitted.
+- **Failed retry interval**: The time, in cents, interval for resubmitting the task after a failed task.
+- **Delayed execution time**: the time, in cents, that a task is delayed in execution.
+- **Timeout alarm**: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
+- **Program type**: supports Java、Scala and Python.
```
Comment (zhongjiajie) [inline docs/en-us/2.0.3/user_doc/guide/task/spark.md:24]:
```suggestion
- **Program type**: supports Java, Scala and Python.
```

**Review by zhongjiajie — COMMENTED** [review]
And NIT, others LGTM and thanks @QuakeWang

File: docs/en-us/2.0.3/user_doc/guide/task/spark.md (hunk 1 above)
Comment (QuakeWang) [reply docs/en-us/2.0.3/user_doc/guide/task/spark.md]: > 

I have finished revising it. : )

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks~

**Pull request merged**